CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+### backend-gl-0.6.0 (2-11-2020)
+  - Re-enable the GL backend for Linux, Mac, and Web
```

Comment:
sorry about being so nitpicky... the other notes start with small letters, not capitals, and we should be consistent with them

Reply:
Oops. That makes sense.